nemo/collections/nlp/data/question_answering_squad/qa_squad_processing.py

```diff
@@ -1,6 +1,6 @@
-# Copyright 2020 NVIDIA. All Rights Reserved.
 # Copyright 2018 The Google AI Language Team Authors and
 # The HuggingFace Inc. team.
```

Comment:
Similarly here...

Reply:
From a very quick glance, transformers is also apache 2, so do we copy the entire apache license again?